op/model.py

```diff
@@ -186,8 +188,17 @@ def __repr__(self):
 class RelationData(Mapping):
     def __init__(self, relation, our_unit, backend):
         self.relation = weakref.proxy(relation)
-        self._data = {our_unit: RelationUnitData(self.relation, our_unit, True, backend)}
-        self._data.update({unit: RelationUnitData(self.relation, unit, False, backend) for unit in self.relation.units})
+        self._data = {our_unit: RelationDataContent(self.relation, our_unit, (lambda _: True), backend)}
```

Comment:
The lambda here escaped me earlier. This looks a bit strange at first sight.

Reply:
+1, answered in https://github.com/canonical/operator/pull/47#discussion_r352296886